src/client/datascience/ipywidgets/webviewIPyWidgetCoordinator.ts

```diff
@@ -71,5 +71,6 @@ export class WebviewIPyWidgetCoordinator implements IInteractiveWindowListener {
                 this.postEmitter.fire(e);
             }
         });
+        this.messageCoordinator.initialize();
```

Comment:
I believe this needs to be a return
```suggestion
        return this.messageCoordinator.initialize();
```

Reply:
Oops, didn't see as a suggestion, fixed